meta-lxatac-software/recipes-connectivity/networkmanager/networkmanager_%.bbappend

```diff
@@ -0,0 +1,3 @@
+# The terminal user interface is the one we document in the manual.
+# Make sure it is installed.
+PACKAGECONFIG:append = "nmtui"
```

Comment:
Seems as if the upstream change left `-Dnmtui=true` in the `EXTRA_OEMESON` . Thus potentially a patch opportunity. This change is fine however.